server/src/test/java/org/elasticsearch/search/aggregations/bucket/range/RangeAggregatorTests.java

```diff
@@ -146,7 +146,8 @@ public void testUnboundedRanges() throws IOException {
                 null,
                 Collections.emptyMap(),
                 null,
-                false
+                false,
+                null
```

Comment:
So so so many parameters. Nothing you can do about it now, but we should think about something cleaner eventually.

Reply:
I agree it's ugly. I can create a new constructor with default values (`false` and `null`) for the time series parameters. WDYT?

Reply:
I'm not at all sure! Maybe a builder or some test utility methods or something.

I don't think it's a problem for this PR though. It seems like a great candidate for a follow up change.